### PR TITLE
docs: translate cloneElement page

### LINKED ISSUE #547: `cloneElement`
Page: https://id.react.dev/reference/react/cloneElement

### PR DESCRIPTION
**Pull request by rianonehub**
Closes #547

## Description

Translate the **cloneElement** page.
Page URL: https://id.react.dev/reference/react/cloneElement

### CODE REVIEW

File: src/content/reference/react/cloneElement.md [hunk 1]
```diff
@@ -43,42 +43,42 @@ const clonedElement = cloneElement(
...
-* `cloneElement` makes it harder to trace the data flow, so **try the [alternatives](#alternatives) instead.**
+* `cloneElement` membuat pelacakan aliran data lebih sulit, jadi **cobalah beberapa [alternatif](#alternatives) sebagai gantinya.**
 
 ---
 
-## Usage {/*usage*/}
+## Penggunaan {/*usage*/}
 
-### Overriding props of an element {/*overriding-props-of-an-element*/}
+### Menimpa _props_ dari suatu elemen {/*overriding-props-of-an-element*/}
```
Comment (r17x) [inline src/content/reference/react/cloneElement.md:79]: don't use *italic* in **Heading**

**Review by r17x — DISMISSED** [review]
1. Don' use italic in Heading
2. for consistency adn same with other translation, Please use `*` instead of `_`.

**Comment by rianonehub** [comment]
> 1. Don' use italic in Heading
> 2. for consistency adn same with other translation, Please use `*` instead of `_`.

Hi @r17x 
Thanks for the correction, will fix it.

**Comment by mhaidarhanif** [comment]
@rianonehub has resolved the issues, @r17x can check and approve if everything's okay

**Review by mazipan — APPROVED** [review]

**Comment by mhaidarhanif** [comment]
@mazipan @resir014 All changes approved and as @r17x might be busy until 18 June so the last requested has been dismissed.

**Pull request merged**